pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```diff
@@ -0,0 +1,45 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkgconfig, go, gnome3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+
+  nativeBuildInputs = [ pkgconfig go ];
+  
+  buildInputs = [ gnome3.glib gnome3.gtk libhandy ];
+
+  desktopFile = makeDesktopItem {
+		name = pname;
+    desktopName = "gtkcord";
+		exec = "gtkcord3";
+		icon = "gtkcord3";
+		categories = "GTK;GNOME;Chat;";
+	};
+
+	preFixup = ''
+		mkdir -p $out/share/icons/hicolor/256x256/apps/ $out/share/applications/
+		# Install the desktop file
+		cp "${desktopFile}"/share/applications/* $out/share/applications/
+		# Install the icon
+		cp "$src/logo.png" $out/share/icons/hicolor/256x256/apps/gtkcord3.png
+	'';
+
+  vendorSha256 = "0kzq8x0qcjr443ydj05s2f8ybszc2jzqa2annadp3hry8x2va2in";
+
+  subPackages = [ "." ];
+
+  meta = with lib; {
+    description = "A lightweight Discord client which uses GTK3 for the user interface.";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ skykanin ];
+    homepage = "https://github.com/diamondburned/gtkcord3";
+    platforms = platforms.x86_64;
```

Comment:
This does not look correct. Missing Linux or Darwin qualifier.

Reply:
Oh, `platforms.x86_64` does actually exist. Sorry about that. Though it is usually better to just use `platforms.unix` since `aarch64` is on the rise.